demo_exam/Task1.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+int getLength(char str[]) {
+	int i = 0;
+	while (str[i]) {
+		i++;
+	}
+	return i;
+}
+
+bool isHighMonth(char str[]) {
+	if ((str[3] == '1' && str[4] != '1') || str[3] == '3' || str[3] == '5' || str[3] == '7' ||
```

Comment:
Why do you check if str[3] is 3, 5, 7 or 9? If you have one of these months, the date should look like 30.05.